src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000040> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000723>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000041> (brine channel)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://askabiologist.asu.edu/brine-channels") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000041> "A channel through which brine flows.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000041> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000041> "brine channel"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> <http://purl.obolibrary.org/obo/ENVO_00000395>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000042>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000042> (brine channel formation process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000042> "A material accumulation process during which trapped brine droplets, between ice crystals formed during seawater freezing, form brine channels.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000042> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000042> "brine channel formation process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> <http://purl.obolibrary.org/obo/ENVO_03000009>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01000277>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000042> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_03000041>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000043> (material transformation process)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000043> "A process during which the one or more materials, present within a given site, transform into another one or more materials.")
```

Comment:
@pbuttigieg I've taken your definiton.

I've added comment:
> A material transformation process only refers to ENVO:environmental material classes (e.g. bulk and typically impure substances), rather than things like the conversion of a specific chemical into another. 

editors note:
> A different material transformation process class (or similarly named class) pertaining to the conversion of a specific chemical into another belongs in CHEBI and or REX.